presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -234,6 +234,8 @@
 
     private boolean pushRemoteExchangeThroughGroupId;
     private boolean isOptimizeMultipleApproxPercentileOnSameFieldEnabled = true;
+    private boolean prestoOnSparkNativeEngineExecutionEnabled;
+    private String prestoOnSparkNativeEngineExecutionExecutablePath;
```

Comment:
- remove "presto on spark"; the main engine does not know about spark. Also, drop the word engine; it's kinda redundant. Same for everywhere else in the PR. Just call them `nativeExecutionXXX`
- you will need to fix the unit test as well. All the presto configs have unit tests

Reply:
Dumb question: These properties seem to be only Spark related. Why would they sit in presto properties?

Reply:
@tanjialiang the reason is we need to access these configs inside the Presto optimizer/plan fragmenter (same as the Presto-on-Spark java path) which force us to make them as Presto properties.